.travis.yml

```diff
@@ -1,7 +1,22 @@
 language: go
 
+matrix:
+  include:
+  - name: amd64
+    os: linux
+    env: ARCH=amd64
+  - name: ppc64le
+    os: linux-ppc64le
+    env: ARCH=ppc64le
+
 before_install:
   - sudo apt-get update -qq
+  - sudo apt-get remove docker-ce
```

Comment:
why ``remove`` and ``install`` docker-ce?

Reply:
we need the docker version 18.09.0 or later which is supporting `docker manifest`, but the docker version provided from Travis is a bit older

Reply:
gotcha, can this be done with single cmd similar to ``apt-get upgrade docker-ce``? I barely use ubuntu, please forgive me if it's a silly question:-)